saplings/contracts/public/mainfest.json

```diff
@@ -0,0 +1,9 @@
+{
+  "short_name": "Contracts",
+  "name": "Contracts sapling",
+  "start_url": ".",
+  "display": "standalone",
+  "theme_color": "#000000",
+  "background_color": "#ffffff"
+}
+  
```

Comment:
This file is missing a newline at the end.

Reply:
Now it looks like there's an extra line here.  This might be a windows vs *nix issue.  

(Also, let the original commenters resolve their issues - that way you know their request has been satisfied).